bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/parallacticangle/ParallacticAngleControls.scala

```diff
@@ -259,22 +259,23 @@ class ParallacticAngleControls(isPaUi: Boolean) extends GridBagPanel with Publis
       dialog.visible = true
       updateSchedulingBlock(dialog.schedulingBlock)
     }
+  }
 
 
   /**
    * This should be called whenever the position angle changes to compare it to the parallactic angle.
    * A warning icon is displayed if the two are different. This is a consequence of allowing the user to
    * set the PA to something other than the parallactic angle, even when it is selected.
    */
-  def positionAngleChanged(positionAngleText: String): Unit = {
+  def positionAngleChanged(positionAngleText: String): Unit = Swing.onEDT {
     // We only do this if the parallactic angle can be calculated, and is different from the PA.
     for {
       e     <- editor
       angle <- parallacticAngle
       fmt   <- formatter
     } {
-      val explicitlySet = !fmt.format(ParallacticAngleControls.angleToDegrees(angle)).equals(positionAngleText) &&
-                          !fmt.format(ParallacticAngleControls.angleToDegrees(angle + Angle.fromDegrees(180))).equals(positionAngleText)
+      val explicitlySet = !fmt.format(angle.toDegrees).equals(positionAngleText) &&
```

Comment:
=grins=
I think we had this exact same discussion before when I made a PR specifically regarding this.
There were several reasons that we did this as text: one because the input was coming from a text field and we wanted to indicate any differences between the angle and text field and not just numerical differences, and the other because the actual parallactic angle is at an extremely high precision, so we use the formatter to chop it to two decimal places.